doc/classes/SceneTree.xml

```diff
@@ -215,6 +221,13 @@
 				To handle 'Go Back' button when this option is disabled, use [constant MainLoop.NOTIFICATION_WM_GO_BACK_REQUEST].
 			</description>
 		</method>
+		<method name="set_scene_tree_physics_interpolation_enabled">
```

Comment:
Have done this, called `scene_tree_physics_interpolation`.

Wasn't sure about whether to have `scene_tree` as part of the name as it is part of `SceneTree`, so that kind of follows, but we are ending up with quite a few things called `physics_interpolation` and maybe something like this would disambiguate it.

Let me know if e.g. `physics_interpolation` is preferred, or something better.

Reply:
Yeah I think `scene_tree` is redundant when we are in the `SceneTree` class, both for the property name and the functions. It's implicit that all methods and properties of `SceneTree` impact the whole tree.

Reply:
Renamed to `physics_interpolation` / `physics_interpolation_enabled`.